at_secondary/at_secondary_server/lib/src/verb/handler/sync_progressive_verb_handler.dart

```diff
@@ -79,49 +79,14 @@ class SyncProgressiveVerbHandler extends AbstractVerbHandler {
   Map _populateMetadata(value) {
     var metaDataMap = <String, dynamic>{};
     AtMetaData? metaData = value?.metaData;
-    if (metaData != null) {
-      if (metaData.ttl != null) {
-        metaDataMap.putIfAbsent(AT_TTL, () => metaData.ttl.toString());
-      }
-      if (metaData.ttb != null) {
-        metaDataMap.putIfAbsent(AT_TTB, () => metaData.ttb.toString());
-      }
-      if (metaData.ttr != null) {
-        metaDataMap.putIfAbsent(AT_TTR, () => metaData.ttr.toString());
-      }
-      if (metaData.isCascade != null) {
-        metaDataMap.putIfAbsent(CCD, () => metaData.isCascade.toString());
-      }
-
-      if (metaData.dataSignature != null) {
-        metaDataMap.putIfAbsent(
-            PUBLIC_DATA_SIGNATURE, () => metaData.dataSignature.toString());
-      }
-      if (metaData.isBinary != null) {
-        metaDataMap.putIfAbsent(IS_BINARY, () => metaData.isBinary.toString());
-      }
-      if (metaData.isEncrypted != null) {
-        metaDataMap.putIfAbsent(
-            IS_ENCRYPTED, () => metaData.isEncrypted.toString());
-      }
-
-      if (metaData.createdAt != null) {
-        metaDataMap.putIfAbsent(
-            CREATED_AT, () => metaData.createdAt.toString());
-      }
-      if (metaData.updatedAt != null) {
-        metaDataMap.putIfAbsent(
-            UPDATED_AT, () => metaData.updatedAt.toString());
-      }
-      if (metaData.sharedKeyEnc != null) {
-        metaDataMap.putIfAbsent(
-            SHARED_KEY_ENCRYPTED, () => metaData.sharedKeyEnc);
-      }
-      if (metaData.pubKeyCS != null) {
-        metaDataMap.putIfAbsent(
-            SHARED_WITH_PUBLIC_KEY_CHECK_SUM, () => metaData.pubKeyCS);
-      }
+    if (metaData == null) {
+      return metaDataMap;
     }
+    metaData.toJson().forEach((key, value) {
+      if (value != null) {
+        metaDataMap.putIfAbsent(key, () => value.toString());
```

Comment:
I think you can just do `put` here rather than `putIfAbsent`

Reply:
The map does not have put method hence changed the above line as follows

```dart
metaData.toJson().forEach((key, value) {
      if (value != null) {
        metaDataMap[key] = [value];
      }
    });
`